hydra/benchmark.sh

```diff
@@ -2,34 +2,11 @@
 set -euo pipefail
 IFS=$'\n\t'
 
-# sets up PG14, configs, and build deps on ubuntu 22.04
-sudo ./install.sh
-
-# download hits.tsv if we dont already have it
-if [ ! -e hits.tsv ]; then
-    wget --no-verbose --continue 'https://datasets.clickhouse.com/hits_compatible/hits.tsv.gz'
-    gzip -d hits.tsv.gz
-fi
-
-# ensure postgres user can access hits.tsv
-chmod a+rX . hits.tsv
-
-# set up db, user, table; load data.
-sudo -u postgres psql -t -c 'CREATE DATABASE test'
-sudo -u postgres psql test -t < create.sql
-sudo -u postgres psql test -t -c 'TRUNCATE hits'
-sudo -u postgres psql test -t -c '\timing' -c "\\copy hits FROM 'hits.tsv'"
-
-# COPY 99997497
-# Time: 1320024.670 ms (22:00.025)
+# load data
```

Comment:
For this to work, the postgres client needs to be installed. Please grep `apt install` in the ClickBench repository for other databases which install stuff.

Reply:
hmm, ok it seems the pattern is to leave a commented-out command for these.

Reply:
updated.